### PR TITLE
Ingress class namespaced params beta

### PR DESCRIPTION
**Pull request by hbagdi**
Enhancement issue: https://github.com/kubernetes/enhancements/issues/2365
k/k PR: https://github.com/kubernetes/kubernetes/pull/101711

### CODE REVIEW

**Comment by hbagdi** [comment]
Putting hold until the k/k PR is merged in.
/hold

**Comment by PI-Victor** [comment]
thank you for opening this!

/assign 
/milestone 1.22
/sig network

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (PI-Victor) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:127]: please leave this line in place and add the `Beta` version below it.

**Review by PI-Victor — CHANGES_REQUESTED** [review]

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md
Comment (hbagdi) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:127]: Updated!

**Comment by hbagdi** [comment]
k/k PR has been merged.
/hold cancel

File: content/en/docs/concepts/services-networking/ingress.md
Comment (sftim) [inline content/en/docs/concepts/services-networking/ingress.md:229]: For graduation to beta, it'd be good to expand this paragraph with some context about why you might want the params for an IngressClass to be namespaced rather than cluster scoped.

Comment (sftim) [reply content/en/docs/concepts/services-networking/ingress.md:229]: (one example: to restrict which namespaces are allowed to use that ingress type).

Comment (hbagdi) [reply content/en/docs/concepts/services-networking/ingress.md:229]: I elaborated to include benefits of namespace-scoped resources. Please take another look, thank you.

File: content/en/docs/concepts/services-networking/ingress.md
Comment (sftim) [inline content/en/docs/concepts/services-networking/ingress.md:236]: nit: CustomResourceDefinition (no spaces)

Comment (hbagdi) [reply content/en/docs/concepts/services-networking/ingress.md]: @sftim updated!

**Review by sftim — COMMENTED** [review]
/lgtm

**Comment by robscott** [comment]
Thanks!

/lgtm

**Comment by PI-Victor** [comment]
👍🏻 
/approve

**Pull request merged**
